test/connectors/MarriageAllowanceDESConnectorSpec.scala

```diff
@@ -18,71 +18,65 @@ package connectors
 
 import com.codahale.metrics.Timer
 import com.github.tomakehurst.wiremock.client.WireMock._
-import connectors.MarriageAllowanceDESConnector.baseUrl
 import errors._
-import metrics.Metrics
+import metrics.TamcMetrics
 import models._
 import org.mockito.ArgumentMatchers
 import org.mockito.ArgumentMatchers.any
-import org.mockito.Mockito.{when, verify, times}
+import org.mockito.Mockito.{times, verify, when, reset => resetMock}
+import org.scalatest.BeforeAndAfterEach
 import org.scalatest.mockito.MockitoSugar
 import org.scalatest.prop.TableDrivenPropertyChecks._
 import org.scalatestplus.play.guice.GuiceOneAppPerSuite
 import play.api.Application
 import play.api.http.Status._
 import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.inject.{Injector, bind}
 import play.api.libs.json.{JsValue, Json}
 import test_utils.WireMockHelper
 import uk.gov.hmrc.domain.{Generator, Nino}
 import uk.gov.hmrc.http._
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
 import uk.gov.hmrc.play.test.UnitSpec
-import utils.WSHttp
+
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
-class MarriageAllowanceDESConnectorTest extends UnitSpec with GuiceOneAppPerSuite with WireMockHelper with MockitoSugar {
+class MarriageAllowanceDESConnectorSpec extends UnitSpec with GuiceOneAppPerSuite with WireMockHelper with MockitoSugar with BeforeAndAfterEach {
+
+  val mockMetrics: TamcMetrics = mock[TamcMetrics]
+  val mockHttp: HttpClient = mock[HttpClient]
```

Comment:
recommened wiremock in the future probs not yet on your todo list thou :)

Reply:
Noted